test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -761,13 +779,15 @@ def return_schema(for_collection, plugin_type='module'):
 
 
 def deprecation_schema(for_collection):
+
     main_fields = {
         Required('why'): doc_string,
-        Required('alternative'): doc_string,
-        Required('removed_from_collection'): collection_name,
-        'removed': Any(True),
+        require_only_one(['alternative', 'alternatives']),
```

Comment:
Don't you still need to keep entries `'alternative': doc_string` and `'alternatives': doc_string` so that the types of these are validated?

Also `require_only_one` cannot be part of a dictionary since it's not a key-value pair.